tests/unit/expire.tcl

```diff
@@ -833,6 +833,24 @@ start_server {tags {"expire"}} {
         assert_equal [r debug set-active-expire 1] {OK}
     } {} {needs:debug}
 
+    test {import-source can be closed when import-mode is off} {
+        r flushall
+        set id [r client id]
+
+        catch {r client import-source on} err
+        assert_match {*Server is not in import mode and we only allow to close import-source status*} $err
```

Comment:
We shouldn't match on the exact error messages. Clients are not supposed to do this either, because it can change. The first word is a code that indicates the class of error and this is what clients should look at.

```suggestion
        assert_match {ERR*} $err
```